### PR TITLE
tile center

### PR DESCRIPTION
**Pull request by beig**
added center/centre image in tile

### CODE REVIEW

File: lib/output.js [hunk 1]
```diff
@@ -726,6 +727,14 @@ function tile (options) {
     } else if (is.defined(options.layout) && options.layout === 'google') {
       this.options.tileSkipBlanks = 5;
     }
+    // Center image in tile
+    if (is.defined(options.centre) || is.defined(options.center)) {
+      if (is.bool(options.centre) || is.bool(options.center)) {
+        this.options.centre = options.centre || options.center;
```
Comment (lovell) [inline lib/output.js:733]: Please can you use `this._setBooleanOption()` to set this boolean property, preferably with a `tile` prefix for consistency, i.e. `tileCentre`. You may also want to set it to a default value of `false` when `this.options` is created in the constructor.

File: src/pipeline.cc [hunk 2]
```diff
@@ -1404,6 +1405,10 @@ Napi::Value pipeline(const Napi::CallbackInfo& info) {
     vips_enum_from_nick(nullptr, VIPS_TYPE_FOREIGN_DZ_DEPTH,
     sharp::AttrAsStr(options, "tileDepth").data()));
 
+  if (sharp::HasAttr(options, "centre")) {
+    baton->centre = sharp::AttrAsBool(options, "centre");
```
Comment (lovell) [inline src/pipeline.cc:1409]: The `if` wrapper can be removed by ensuring `this.options.tileCentre` defaults to `false` in JS land.

**Review by lovell — CHANGES_REQUESTED** [review]
Hi, thank you for this PR, I've left a couple of comments inline that should help you with those failing tests.

**Comment by beig** [comment]
I hopefully correctly applied your suggestions

**Comment by lovell** [comment]
Thank you for the update, it looks like there is a linting error to deal with.

**Comment by lovell** [comment]
Brilliant, thank you very much, this will be in v0.26.2.

**Pull request merged**